android/app/src/main/AndroidManifest.xml

```diff
@@ -1,13 +1,17 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.lndr"
     android:versionCode="1"
     android:versionName="1.0">
 
     <uses-feature android:name="android.hardware.camera" />
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" /> 
-    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
+<!-- NOTE: this line does not appear to work -->    
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" tools:remove="${excludeSystemAlertWindowPermission}" -->
```

Comment:
the end of the line should be changed from `-->` to `/>`

Reply:
Where did you get that? Comments end in `-->` AFAIK

https://www.w3schools.com/xml/xml_syntax.asp

However, I don't want these comments to stay in here at all.

Reply:
nice catch!